pkg/network/netlink/conntracker_integration_test.go

```diff
@@ -132,8 +132,10 @@ func testMessageDump(t *testing.T, f *os.File, serverIP, clientIP net.IP) {
 	defer udpServer.Close()
 
 	for i := 0; i < 100; i++ {
-		nettestutil.PingTCP(t, clientIP, natPort)
-		nettestutil.PingUDP(t, clientIP, nonNatPort)
+		tc := nettestutil.PingTCP(t, clientIP, natPort)
+		tc.Close()
```

Comment:
Are we not closing twice?
It is being done in the function itself as well.

Reply:
We may Close twice, but that is not a problem.